sycl/test/basic_tests/vectors/ctad_fail.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -std=c++17 -fsyntax-only -Xclang -verify %s -I %sycl_include/sycl -Xclang -verify-ignore-unexpected
+// RUN: %clangxx -fsyntax-only -Xclang -verify %s -fsycl -fsycl-targets=%sycl_triple -Xclang -verify-ignore-unexpected
```

Comment:
How is this change related to moving files to libpi directory?
NOTE: I don't think we should make this change. Original test validates compilation of DPC++ header with C++ compiler, but this change involves device compilation as well.